tests/github/interactive_api/experiments/tensorflow_mnist/experiment.py

```diff
@@ -0,0 +1,72 @@
+import logging
+
+import tensorflow as tf
+
+from openfl.interface.interactive_api.experiment import ModelInterface, FLExperiment
+from openfl.interface.interactive_api.federation import Federation
+from tests.github.interactive_api.experiment_runner import run_experiment
+from tests.github.interactive_api.experiments.tensorflow_mnist.settings import model, optimizer, X_train, y_train, X_valid, y_valid, batch_size
```

Comment:
This imports should be split. @itrushkin flake8 run for this file/folder?

Reply:
`tests/github/interactive_api` folder is excluded in `setup.cfg`